cloudinit/sources/DataSourceEc2.py

```diff
@@ -125,12 +126,15 @@ def _get_data(self):
                 LOG.debug("FreeBSD doesn't support running dhclient with -sf")
                 return False
             try:
-                with EphemeralDHCPv4(self.fallback_interface):
+                with EphemeralIPNetwork(
+                    self.fallback_interface, ipv6=True
```

Comment:
I'm on the fence whether to make ipv6 default to False (as it does now), which makes `EphemeralIPNetwork` more of a drop-in replacement for `EphemeralDHCPv4()`, or make ipv6 default to True, on the assumption that the new context manager will only be used where dual-stack is expected and is therefore a better default. I'm leaning towards option 2 but no strong preference. Thoughts?

Reply:
I lean towards option 1 just so a common thing can be imported everywhere, but same...no strong preference